content/en/docs/examples/virtual-machines/index.md

```diff
@@ -69,7 +69,7 @@ You can find details of configuring MySQL at [Mysql](https://mariadb.com/kb/en/l
 On the VM add ratings database to mysql.
```

Comment:
Maybe (and I know this wasn't changed), `On the VM, add a ratings database to mysql`.